src/actionCreators/resources.js

```diff
@@ -37,7 +37,7 @@ import {
 } from "actionCreators/user"
 import { addResourceHistory } from "actionCreators/history"
 import _ from "lodash"
-import { setCurrentComponent } from "actions/index"
+import setCurrentComponent from "actions/index"
```

Comment:
`setCurrentComponent` became a default export so this simple change is in a number of files.